src/app/beer_garden/api/http/handlers/v1/command_publishing_blocklist.py

```diff
@@ -0,0 +1,140 @@
+        responses:
+          204:
+            description: Command has been successfully removed from block list
+            schema:
+              $ref: '#/definitions/CommandPublishingBlocklist'
+          404:
+            $ref: '#/definitions/404Error'
+          50x:
+            $ref: '#/definitions/50xError'
+        tags:
+          - Command Block List
+        """
+        blocked_command = CommandPublishingBlockList.objects.get(
+            id=command_publishing_id
+        )
+        systems = System.objects(
+            namespace=blocked_command["namespace"], name=blocked_command["system"]
+        )
+        self.get_or_raise(System, SYSTEM_UPDATE, id=systems[0].id)
+
+        if config.get("garden.name") != blocked_command["namespace"]:
+            beer_garden.router.route(
+                Operation(
+                    operation_type="COMMAND_BLOCK_LIST_REMOVE",
+                    args=[command_publishing_id],
+                    target_garden_name=config.get("garden.name"),
+                )
+            )
+        beer_garden.router.route(
+            Operation(
+                operation_type="COMMAND_BLOCK_LIST_REMOVE",
+                args=[command_publishing_id],
+                target_garden_name=blocked_command["namespace"],
+            )
+        )
+
+        self.set_status(204)
+
+
+class CommandPublishingBlockListAPI(AuthorizationHandler):
+    def get(self):
+        """
+        ---
+        summary: Retrieve list of commands in publishing block list
+        responses:
+          201:
+            description: list of commands in publishing block list
+            schema:
+              $ref: '#/definitions/CommandPublishingBlocklist'
+          400:
+            $ref: '#/definitions/400Error'
+          50x:
+            $ref: '#/definitions/50xError'
+        tags:
+          - Command Block List
+        """
+        response = beer_garden.router.route(
+            Operation(
+                operation_type="COMMAND_BLOCK_LIST_GET",
+            )
+        )
+
+        self.write(response)
+
+    def post(self):
+        """
+        ---
+        summary: Add a list of commands to event publishing block list
+        parameters:
+          - name: CommandPublishingBlocklist
+            in: body
+            description: The system, namespace and command name
+            schema:
+              type: array
+              items:
+                  $ref: '#/definitions/CommandPublishingBlocklist'
+        consumes:
+          - application/json
+        responses:
+          201:
+            description: Command has been added to publishing block list
+            schema:
+              $ref: '#/definitions/CommandPublishingBlocklist'
+          400:
+            $ref: '#/definitions/400Error'
+          50x:
+            $ref: '#/definitions/50xError'
+        tags:
+          - Command Block List
+        """
+        errors = []
+        for command in self.request_body:
```

Comment:
You'll want to use the marshmallow schemas to do input validation.  That will give you a clean way to check for required fields and such, and then you can catch the ValidationError from marshmallow if it's raised and reraise the appropriate exception from `beer_garden.api.http.exceptions`.  That will get you the appropriate HTTP response for validation errors.

There is also a `schema_validated_body` helper available that may or may not be useful for this.